src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -363,20 +363,25 @@ void NotationViewInputController::endOfScore()
 
 void NotationViewInputController::wheelEvent(QWheelEvent* event)
 {
-    QPoint pixelsScrolled = event->pixelDelta();
-    QPoint stepsScrolled = event->angleDelta();
-
     int dx = 0;
     int dy = 0;
     qreal stepsX = 0.0;
     qreal stepsY = 0.0;
 
+    QPoint stepsScrolled = event->angleDelta();
+
+// pixelDelta is unreliable on X11
```

Comment:
Linux is not only about X11. Will this fix work fine on Wayland?

Reply:
Are we building MuseScore for supporting Wayland? I don't think we do.

Reply:
I tested this on wayland with both KDE and sway and the pixelDelta value isn't provided there, meaning the problem doesn't arise in the first place _for now_. It could be that in the future wayland will provide that in a compliant way, in which case the exception should be limited to X11 only. It could also end up broken as with X11 right now. 

For now the change doesn't do anything on Wayland, but maybe this would need to be reevaluated in the future.

Reply:
But whether it is X11 or Wayland is a runtime thing rather than a compile time one, isn't it?

Reply:
Yeah, the same binary runs on X11 and Wayland, so that would have to be done at runtime. I believe the best way to do that is by looking at certain environment variables.

Reply:
I guess that'd be `$XDG_SESSION_TYPE`, which should be `x11`or `wayland` Or not used at all on non-Linux/UNIX platforms?

Reply:
Or `$DISPLAY` vs. `$WAYLAND_DISPLAY` vs. neither

Reply:
I now put up a version that does not affect Wayland by checking `$WAYLAND_DISPLAY` that should be safe for most situations. I still have it dependent on `Q_OS_LINUX` so as to not make any assumptions about the environment variables on other platforms.